client/src/modules/journal/modals/trialBalanceMain.body.html

```diff
@@ -28,6 +15,17 @@
       <span translate>FORM.BUTTONS.VIEW_ERRORS</span>
     </button>
   </div>
+
+  <div class="col-xs-6 text-right">
+    <button
+      id="export"
+      type="button"
```

Comment:
I wanted to comment on this type being redundant on a button but: 
> Tip: Always specify the type attribute for the <button> element. Different browsers may use different default types for the <button> element.
[source](https://www.w3schools.com/tags/att_button_type.asp)